### ISSUE TITLE
Creating Garchen Rinpoche STT dataset with latest transcribed data.

### ISSUE THREAD

**Issue by gangagyatso4364** [OpenPecha/garchen-rinpoche-speech-sig#49]
### Description 
We have around 30 hours of Garchen Rinpoche training data that has been transcribed by our annotators. 
In this repo we only keep Garchen Rinpoche transcribed transcript only and reviewed transcript are not included as at this stage not finished reviewing the data.
 
The repo will have train, val, test set. 

### Completion Criteria 
A Hugging face dataset repo ready for training and benchmark evaluation.

### Subtask
- [x] extract transcribed data from database 
- [x] filter any non tibetan characters or unwanted repeated chars 
- [x] create train data split 
- [x] create validation and test set for Garchen rinpoche within same dataset repo.

reviewer
- [x] @dhakar66

**Comment by tenkus47** [comment]
need some more work to finish this card. will be finished by end of day.

**Comment by gangagyatso4364** [comment]
[Unbiased Benchmark](https://huggingface.co/datasets/ganga4364/garchen_rinpoche_unbiased_benchmark)
[training data 29 hours](https://huggingface.co/datasets/ganga4364/garchen_rinpoche_data_latest_29hours)
